lib/cylc/dump.py

```diff
@@ -32,8 +31,6 @@ def get_stop_state_summary(lines):
     for line in list(lines):
         if line.startswith('Remote command'):
             lines.remove(line)
-    line0 = lines.pop(0)
```

Comment:
Taking a better look at the following lines this seems OK as the code scans through lines looking for the pattern `time :`.

```python
line1 = lines.pop(0)
while not line1.startswith("time :"):
    line1 = lines.pop(0)
```